README.md

```diff
@@ -226,6 +228,100 @@ configuration steps, behavior, and precedence are outlined [here](https://github
 and requesting static metadata. This section outlines the common use cases of AEMM; advanced usage and behavior are documented [here](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/docs/usage.md).
 
+## Auto Scaling
+To view the available flags for the Auto Scaling command use `autoscaling --help`:
+```
+Mock autoscaling information
+
+Usage:
+  ec2-metadata-mock autoscaling [flags]
+  ec2-metadata-mock autoscaling [command]
+
+Available Commands:
+  target-lifecycle-state Mock autoscaling lifecycle state transitions
+
+Flags:
+  -h, --help   help for autoscaling
+
+Global Flags:
+  -c, --config-file string              config file for cli input parameters in json format (default: $HOME/aemm-config.json)
+  -n, --hostname string                 the HTTP hostname for the mock url (default: 0.0.0.0)
+  -I, --imdsv2                          whether to enable IMDSv2 only, requiring a session token when submitting requests (default: false, meaning both IMDS v1 and v2 are enabled)
+  -d, --mock-delay-sec int              spot itn delay in seconds, relative to the application start time (default: 0 seconds)
+  -x, --mock-ip-count int               number of IPs in a cluster that can receive a Spot Interrupt Notice and/or Scheduled Event (default 2)
+      --mock-trigger-time string        spot itn trigger time in RFC3339 format. This takes priority over mock-delay-sec (default: none)
+  -p, --port string                     the HTTP port where the mock runs (default: 1338)
+      --rebalance-delay-sec int         rebalance rec delay in seconds, relative to the application start time (default: 0 seconds)
+      --rebalance-trigger-time string   rebalance rec trigger time in RFC3339 format. This takes priority over rebalance-delay-sec (default: none)
+  -s, --save-config-to-file             whether to save processed config from all input sources in .ec2-metadata-mock/.aemm-config-used.json in $HOME or working dir, if homedir is not found (default: false)
+
+Use "ec2-metadata-mock autoscaling [command] --help" for more information about a command.
+```
+
+### Auto Scaling Target Lifecycle State
+To view the available flags for the Auto Scaling Target Lifecycle State command use `autoscaling target-lifecycle-state --help`:
+```
+Mock autoscaling lifecycle state transitions.
+
+Target states may be changed at a specified time or after a delay, e.g. "InService 2m Terminated" will initially return the InService target lifecycle state then after 2 minutes will return Terminated.
+Valid states are Detached, InService, Standby, Terminated, Warmed:Hibernated, Warmed:Running, Warmed:Stopped, Warmed:Terminated
+Times must be in RFC 3339 format, e.g. "2018-09-01T12:00:00Z".
+Delays are specified as one or more numbers and suffixes, e.g. "2m30s". Valid suffixes are "h", "m", "s", "ms", "us", "ns".
```

Comment:
"delay" is not the name of a flag. It's just the description of an argument value.

Reply:
Sure, I realize that. I know the format of my suggestions is misleading :D 
I meant to say, since AEMM already has a concept of delays, does it make sense to use `time to transition`  or `time between states`  instead of calling this a "delay"? If you don't see any reason for confusion here, then feel free to ignore my suggestion.